### PR TITLE
Change Windows venv lib scheme

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

**Comment by jgraham** [comment]
Fixes #40300 

See https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=101609&view=results for this running on a trigger branch.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Looks like it was copypasta. I'm not merging, it looks like Edge is running past the point of the previous failure, but I'll leave it to you how much to check before landing.

**Pull request merged**
